app/views/admin/protocol_drugs/index.html.erb

```diff
@@ -18,11 +18,16 @@
         <td><%= protocol_drug.dosage %></td>
         <td><%= protocol_drug.rxnorm_code %></td>
         <td><%= protocol_drug.protocol %></td>
-        <% if policy([:manage, protocol_drug]).edit? %>
+        <% if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) %>
```

Comment:
There might be a bug here. Doesn't this bit mean once we enable the flipper flag, everyone will see links?

Reply:
_ditto_